snowflake/Integration.txt

```diff
@@ -0,0 +1,46 @@
+-- creating database and warehouse --
+
+use role sysadmin;
+
+create database if not exists playdb;
+
+create warehouse if not exists playwh with warehouse_size='xsmall' auto_suspend=60 initially_suspended=true;
+
+use schema playdb.public;
+
+-- cloud storage integration --
+
+use role accountadmin;
+
+create storage integration azure_snowflake_integration_171_2
```

Comment:
Name should be as descriptive as possible. In this case `azure_integration` will be enough:
- we don't need those numbers (you can have it for testing, but not for the final version)
- it's obvious, that the integration is between Snowflake and something